pint_server/pint_db_migrate/versions/b381843009e0_trying_enums.py

```diff
@@ -0,0 +1,54 @@
+"""Trying Enums
+
+Revision ID: b381843009e0
+Revises:
+Create Date: 2021-07-30 00:15:20.918363
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'b381843009e0'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+tablenames = ['amazonservers', 'googleservers', 'microsoftservers']
+
+def upgrade():
+    # manual commands
+    op.execute("CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";")
+    for tablename in tablenames:
+        op.execute("alter table {} drop constraint {}_pkey;".format(tablename, tablename))
```

Comment:
suggest using

`op.drop_constraint('%s_pkey' % (tablename), tablename, type_='primary')`

Reply:
is there any difference/benefit in this case between the orm method vs raw SQL ?

Reply:
ORM is supposed to be database agnostic. But we are using postgresql data types directly so it's probably a moot point. But still, we should try to avoid raw SQL if we can.